contents/handbook/small-teams/growth/index.mdx

```diff
@@ -11,27 +11,29 @@ hideAnchor: false
 
 ## Mission
 
-PostHog's company growth targets are met every quarter
-
-## Q4 2022 Objective
-Start charging for session recordings resulting in a $100K increase in ARR from new customers (joint goal with [Session Recording team](/handbook/small-teams/session-recording))
+The most people possible can get value out of PostHog.
 
 ## What the Growth team does
 
-Each month, the Growth and [Exec team](/handbook/small-teams/exec) carry on a [Growth review session](/handbook/growth/growth-engineering/growth-sessions) to accelerate growth work, as well as define tasks for the Growth team.
-
-The Growth team focuses on Activation metrics, namely:
-
-- Signup activation rate: User has signup to the product -> User has successfully discovered the product's value
-- Card activation rate: User has inserted their card details
-- Paid activation rate: User has converted to a paying customer
-
-The team is primarily responsible for the following areas:
-
-- Signup flow
-- Onboarding and integrating PostHog with the customer's product
-- Billing and card activation flows
-- Messaging, emailing and notifications
+- Sign up
+- Activation and reducing time to value
+- Billing and pricing changes
+- Product led growth, including all messaging, emailing and notifications
```

Comment:
I'm not sure we should be responsible for _all_ messaging... Maybe...

```suggestion
- Product led growth, including messaging, emailing and notifications as they relate to the above areas of responsibility
```

Reply:
I would just remove the `all`